qiita_db/support_files/qiita-db.html

```diff
@@ -5722,6 +5723,12 @@
 		<td> bool   DEFO false </td>
 		<td>  </td>
 	</tr>
+	<tr>
+		<td>*</td>
```

Comment:
The patch has a comment, could you make sure that you also add it in the GUI or remove it from both?